### ISSUE TITLE
Spawn Tab tab errors.

### ISSUE THREAD

**Issue by Cuprohastes** [BeipDev/BeipMU#48]
Short version: All the spawn windows with a tab group name will open up in the first tab group that gets spawned, regardless of what name you put in the tab group field.

Testing: 

I created a tab group Tab-A
I added Tab-A to  a set of triggers:
Whispers In (Spawn window name: Psst!)
Whipers Out (Spawn window name: Psst!)
Pages In (Spawn window name: Yo Thar!)
Pages Out (Spawn window name: Yo Thar!)
Error message (Spawn window name: You dun fucked up, boy)
Server messages (Spawn window name: Info)

I generated capturable information:
w me=Test!
p me=Test
wghwajhgroawhg
spoof Testname has connected.

This resulted in a panel called Tab-A with tabs using the names of the spawn windows, as expected.
Dragging the tab group around resulted in the expected bahbviour regarding floating, and snapping to the main window. 

I then changed the Page and Whisper triggers to Tab-B without closing Tab-A.
This resulted in pages and whispers being directed to Tab-A

Then I closed the tabs. A new tab group and spawn windows was not generated on sending of the capturable text. It was also not displayed in the main window

I closed the entire tab group and re-sent my test info.

Whispers and pages opened in Tab-B as expected. 

All other spawns also opened in Tab-B, though they had Tab-A listed as their tab group name.

**Comment by BeipDev** [comment]
Yep, that was the bug I found just a couple hours ago. Awesome to know you'd have found it too!

The problem was I wasn't storing the window title or comparing the titles properly. So the first window created would get all future tabs.
